src/apps/chifra/pkg/names/names.go

```diff
@@ -161,27 +121,29 @@ type NameReader struct {
 	csvReader csv.Reader
 }
 
-func (gr *NameReader) Read() (Name, error) {
+func (gr *NameReader) Read() (types.SimpleName, error) {
```

Comment:
If it's `NameReader`, why is it called `gr`? :)

Reply:
Came from old GitCoin code, so `gr` stands for `grant`. Fixed.